apps/web/src/components/Basenames/RegistrationSearchInput/index.tsx

```diff
@@ -224,8 +227,19 @@ export default function RegistrationSearchInput({
   );
 
   useEffect(() => {
-    setSearchInputFocused(focused);
-  }, [focused, setSearchInputFocused]);
+    const handleScroll = () => {
+      console.log(window.scrollY);
```

Comment:
nit, we should really make this an eslint-config error, sorry

Reply:
🙌 good catch, thanks!